flavors/python-3.6-minimal-EXASOL-6.2.0/flavor_base/language_deps/packages/apt_get_packages

```diff
@@ -2,3 +2,7 @@ ca-certificates|20210119~18.04.2
 python3.6-dev|3.6.9-1~18.04ubuntu1.6
 python3-distutils|3.6.9-1~18.04
 curl|7.58.0-2ubuntu3.16
+git|1:2.17.1-1ubuntu0.9
+libblas-dev|3.7.1-4ubuntu1
+liblapack-dev|3.7.1-4ubuntu1
```

Comment:
Note that both are not required for numpy, but _strongly recommended_:
See [here](https://numpy.org/devdocs/user/building.html#prerequisites)

Reply:
Add comment  to dependency regarding the recommended thing